server/src/main/scala/org/elasticmq/server/QueueMetadataListener.scala

```diff
@@ -0,0 +1,27 @@
+package org.elasticmq.server
+
+import akka.actor.Actor
+import org.elasticmq.QueueData
+import org.elasticmq.actor.queue.{PersistQueue, RemoveQueue, UpdateQueueMetadata}
+
+import scala.collection.mutable
+
+class QueueMetadataListener(storagePath: String) extends Actor {
```

Comment:
The name here should be `QueueConfigStore` or similar, since we're implementing the listener for a specific use-case - not a general-purpose listener